src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Change dependency from `OpenTelemetry.Contrib.Extensions.AWSXRay` to `OpenTelemetry.Extensions.AWS`
+  ([#1289](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1289))
+* Update OpenTelemetry SDK version to `1.5.1` by `OpenTelemetry.Extensions.AWS` package.
```

Comment:
I don't think this is needed for this library's CHANGELOG. This should be covered in the `OpenTelemetry.Extensions.AWS` library's CHANGELOG.